### PR TITLE
Add comment field for sort and copy

### PR DESCRIPTION
**Pull request by lawwm**
Update sort and copy command for comment field.

### CODE REVIEW

**Review by fantablack — COMMENTED** [review]
Nice work! I have tested your code with _basic_ expected and invalid inputs and it appears to be working fine. 

By the way, you might want to add a `Module` parameter in `MESSAGE_USAGE` of `CopyCommand.java`.

File: src/main/java/seedu/address/logic/commands/CopyCommand.java [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_STATUS;
+import static seedu.address.logic.parser.CliSyntax.*;
```
Comment (fantablack) [inline src/main/java/seedu/address/logic/commands/CopyCommand.java:4]: Wildcard import

**Review by fantablack — COMMENTED** [review]
There is an wildcard import in `CopyCommand.java`

**Review by Btaykb — COMMENTED** [review]
LGTM

**Pull request merged**